h3/src/error.rs

```diff
@@ -1,12 +1,16 @@
 //! HTTP/3 Error types
 
-use std::fmt;
+use std::{fmt, sync::Arc};
 
-use crate::{frame, proto, qpack};
+use crate::{frame, proto, qpack, quic};
 
+/// Cause of an error thrown by our own h3 layer
 type Cause = Box<dyn std::error::Error + Send + Sync>;
+/// Error thrown by the underlying QUIC impl
+pub type TransportError = Box<dyn quic::Error>;
```

Comment:
I'd recommend keeping this private too, unless it's needed to be public for something that I didn't notice?

Reply:
Thanks for spotting this, it actually should be `pub(crate)` only.